### ISSUE TITLE
Assault bots on Mimiron hitting 40k+ on tanks

### ISSUE THREAD

**Issue by sannitie** [SunwellTracker/issues#2914]
Decription: 
Assault bots on Mimiron are hitting tanks for more than 40k (see screenshhots in Source) even without the root the mobs do.

How it works: 
Assault bots hits for a lot.

How it should work: 
Assault bots should not hit for that much.

Source (you should point out proofs of your report, please give us some source): 
https://i.imgur.com/4xtrsUV.png
https://i.imgur.com/X5H3TPi.png

**Comment by Cwiercpasiec** [comment]
According to wowhead comments Assault Bots actually were hitting that hard (or even harder) on patch 3.1.0, just after Ulduar release, but they were slowable, stunable, etc.:
https://www.wowhead.com/npc=33350/mimiron#comments
By zapiks44 (15,051 – 5·30·114) on 2009/04/21 (Patch 3.1.0)
[...]
Phase 3 is a similar gear-check for your DPS. Select a ranged DPS (preferably a demo-lock with Soul Link) to tank the ACU. Select another ranged DPS to "kite" the Assault Bots (preferably your highest-damage hunter) while the rest of your DPS kill them as fast as possible. It's also a good idea to have a rogue put Crippling Poison on their offhand and shiv the Assault Bots as quickly as possible, to keep it snared. In case you're wondering why they're kited, lets just say they hit our main tank for 60,000. After dying, the assault bots drop a magnetic core which can be used to bring down the head, stun it, and cause it to take additional damage. If you kill the Assault Bots fast enough, you'll have a good 10-15sec to burn the boss before the next Assault Bot spawns. The junk bots can be easily tanked and AOEed once the phase is over.
[...]

Although such high hits seem like a bug, because later comments mention simply tanking Assault Bots. There is even this one comment:
By Mahtamori (686 – 11) on 2009/05/22 (Patch 3.1.2)
I can't for the life of me figure out why all guides say you should kite the Assault Bots in phase 3. They're perfectly tankable in 25 man, just don't get hit right after they spawn.

And an answer:
Nnoutaja on 2009/05/25 (Patch 3.1.2)
I'm 100% positive that somewhere after 3.1 they put a sneak nerf to assault bots. At your first attempts on Mimiorn they we're oneshotting the OT. Couple weeks later they were hitting for less than the old 10-man versions. The assault bots cast somekind of channeled spell, magnetic field or somethig, which needs to be interrupted, or the OT might be in trouble. 

So in conclusion, it might be intended for Assault Bots to hit that hard, but if so, they should be at least slowable, so we can kite them (or just nerf their dmg a bit).

**Comment by Riztazz** [comment]
Assault bot damage in normal mode has been slightly reduced so on normal mode they shouldnt hit AS hard, they will still hit very hard on hardmode due to emergency mode applied on them.
Snare/slow/stun immunity has been fixed as well
